packages/peregrine/lib/talons/CartPage/ProductListing/EditModal/__tests__/useEditModal.spec.js

```diff
@@ -0,0 +1,36 @@
+import { useAppContext } from '../../../../../context/app';
+import { useEditModal } from '../useEditModal';
+
+jest.mock('../../../../../context/app', () => ({
+    useAppContext: jest.fn()
+}));
+
+describe('return correct open status', () => {
```

Comment:
I personally like using `describe` but I remember I was asked not to use it unless there is no way and you want to mock/perform something for only certain tests.

Reply:
I did find other usages that use it to group tests, so I'm going to leave this for now.